src/PeriodEnd/PaymentsDue/SFA.DAS.ProviderPayments.Calc.PaymentsDue/Infrastructure/Data/Entities/RequiredPaymentEntity.cs

```diff
@@ -1,11 +1,10 @@
 ﻿using System;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
-using SFA.DAS.ProviderPayments.Calc.PaymentsDue.Domain;
 
 namespace SFA.DAS.ProviderPayments.Calc.PaymentsDue.Infrastructure.Data.Entities
 {
-    public class RequiredPaymentEntity : IFundingDue
+    public class RequiredPaymentEntity
     {
         public RequiredPaymentEntity()
         {}
```

Comment:
is it necessary to have the constructor below?

Reply:
Hmmmm. Mine doesn't have a constructor. Hoping that your code is not too far out of date ;)